.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "third_party/googletest"]
 	path = third_party/googletest
 	url = https://github.com/google/googletest.git
+[submodule "partial/extern/RIOT"]
+	path = partial/extern/RIOT
+	url = https://github.com/OYTIS/RIOT.git
```

Comment:
Is it likely that you will be able to upstream your changes? Otherwise it may be better to put this on an `advancedtelematic` fork.